pkg/asset/installconfig/openstack/validation/machinepool.go

```diff
@@ -44,6 +44,9 @@ func ValidateMachinePool(p *openstack.MachinePool, ci *CloudInfo, controlPlane b
 		if p.RootVolume.Size < minimumStorage {
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("rootVolume").Child("size"), p.RootVolume.Size, fmt.Sprintf("Volume size must be greater than %d to use root volumes, had %d", minimumStorage, p.RootVolume.Size)))
 		}
+		if p.RootVolume.Zones != nil {
+			allErrs = append(allErrs, validateZones(p.RootVolume.Zones, ci.Zones, fldPath.Child("rootVolume").Child("zones"))...)
```

Comment:
It seems like `ci.Zones` only contains [compute AZs](https://github.com/openshift/installer/blob/5a12073/pkg/asset/installconfig/openstack/validation/cloudinfo.go#L322), so unless your cinder AZs match your compute AZs, this validation will fail.

It would be nice also to differentiate nova AZs from cinder AZs.

Reply:
right... I think we need to remove this validation, because currently gophercloud doesn't support listing volume availability zones.